src/worker.js

```diff
@@ -131,14 +143,20 @@ this.onmessage = function(e) {
         URL.revokeObjectURL(objectUrl);
       }
 #if MODULARIZE && !MODULARIZE_INSTANCE
+#if MINIMAL_RUNTIME
+      Module = {{{ EXPORT_NAME }}}(imports);
+#else
       Module = {{{ EXPORT_NAME }}}(Module);
 #endif
-      PThread = Module['PThread'];
-      HEAPU32 = Module['HEAPU32'];
-      postMessage({ cmd: 'loaded' });
+#endif
+
+#if !MINIMAL_RUNTIME || !WASM
+      postMessage({ 'cmd': 'loaded' });
```

Comment:
how does minimal runtime avoid the need for `loaded`? and non-WASM?

Reply:
Good point: MINIMAL_RUNTIME always uses async wasm compilation - even in pthreads. So it cannot signal 'loaded' here yet, but it will do that signaling inside the wasm compilation .then() handler. Added a comment to clarify this.